.github/workflows/release.yaml

```diff
@@ -160,6 +161,15 @@ jobs:
           build-args: |
             VERSION=${{ github.event.inputs.version }}
 
+      - name: Build and Push OpenTelemetry Collector with APIClarity plugin
+        uses: docker/build-push-action@v3
+        with:
+          context: plugins
+          tags: otel-apiclarityexporter:${{ github.event.inputs.version }}
+          push: true
+          build-args: |
+            OTEL_COLLECTOR_VERSION=$OTEL_COLLECTOR_VERSION
+
```

Comment:
same comments as in the docker.yaml

Reply:
In [76ce43d](https://github.com/openclarity/apiclarity/pull/255/commits/76ce43d1be9f34c99a5d51006373dc5414768c1c)